src/ui/widgets/XYPlot/xyPlotOptions.test.ts

```diff
@@ -0,0 +1,514 @@
+import { Axis } from "../../../types/axes";
+import { Color } from "../../../types/color";
+import { DType } from "../../../types/dtypes";
+import { Font, FontStyle } from "../../../types/font";
+import { Traces } from "../../../types/traces";
+import {
+  calculateAxisLimits,
+  createAxes,
+  createTraces,
+  roundValue
+} from "./xyPlotOptions";
+
+const DATA = [0, 3, 4, 8];
+const ARRAY_DATA = new DType({
+  arrayValue: new Float32Array(DATA)
+});
+
+describe("round values", (): void => {
+  test("returns zero if values zero", (): void => {
```

Comment:
I was just curious as to why we sometimes use `test()` and other times use `it()` to declare a test. The Jest documentation says that `it()` is an alias for `test()` so functionally they are the same. Some users suggest that `it()` makes the tests more readable. Should we look at picking one and remaining consistent throughout the code?